### ISSUE TITLE
When we are moving cursor before the backslash “\” JAWS reads as Right bracket and for the String "C:\Temp\Scripts\", JAWS is announcing it as "C:\uc1"

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#40537]
GitHubTags:-A11ySev3;-A11yTCS;-A11yMAS;-DesktopApp;-WCAG1.3.1;-DesktopApp;-NVDA;-JAWS;-Narrator-Win32;-Win11;-A11yeDAD; -Visual Studio Code Client
--- Environment Details:​
Application Name: Visual Studio Code
Version: 1.88.0-insider (user setup)
Commit: ea142b5ccdcb797b1de6b1a46fecbf25dea2e229
Date: 2024-03-06T05:47:45.312Z
Electron: 28.2.5
ElectronBuildId: 27336930
Chromium: 120.0.6099.291
Node.js: 18.18.2
V8: 12.0.267.19-electron.0
OS: Windows_NT x64 10.0.22621
Screen reader: Narrator, NVDA2023.3.1, JAWS 2024.2312.53 
--- Repro Steps:

1. Turn on the JAWS.
2. Launch Visual Studio Code
3. Go to the code editor and type a string like "C:\Temp\Scripts".

--- Actual:
When we are moving cursor before the backslash “\” Jaws reads as Right bracket and for the String "C:\Temp\Scripts\", JAWS is announcing it as "C:\uc1".

**Similar issue is not observed with Narrator and JAWS.**
--- Expected:
When we move our cursor before the backslash "\", JAWS should announce backslash.
For the string "C:\Temp\Scripts\", JAWS should announce only"C:\Temp\Scripts\".
--- User Impact:
Screen reader users will get incorrect information about the string present in the code editor.
--- Attachment
[220488_NVDA.zip](https://github-com/microsoft/vscode/files/14508346/220488_NVDA.zip)
[220488_Narrator.zip](https://github-com/microsoft/vscode/files/14508350/220488_Narrator.zip)
[220488_JAWS.zip](https://github-com/microsoft/vscode/files/14508351/220488_JAWS.zip)
